resources/qml/RadioCheckbar.qml

```diff
@@ -0,0 +1,152 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+
+Item
+{
+    id: base
+    property ButtonGroup buttonGroup: null
+
+    property color activeColor: "#3282ff"
```

Comment:
Do those sizes and colors need to be themed?

Reply:
Yup, will fix those.